include/celero/Print.h

```diff
@@ -41,6 +41,7 @@ namespace celero
 		void Failure(const std::string& x);
 		void Run(std::shared_ptr<Result> x);
 		void Done(std::shared_ptr<Result> x);
+		void DoneExperiment(std::shared_ptr<Experiment> x);
```

Comment:
Maybe simple replace total run time in sec with a current run time in sec which is equals to [... us] value?

celero::timer::ConvertSystemTime(x->getRunTime())